config/config-package.py

```diff
@@ -335,6 +345,24 @@ def gha_workflows(self):
         folder = self.path / '.github' / 'workflows'
         folder.mkdir(parents=True, exist_ok=True)
         destination = folder / 'meta.yml'
+        options = self._get_options_for(
+            'github',
+            (
+                'branch',
+                'jobs',
+                'additional_jobs',
```

Comment:
As suggested, we could rename this `extra_lines` 🍀 

```suggestion
                'extra_lines',
```